src/app/dynamic_server/DynamicDispatch.cpp

```diff
@@ -0,0 +1,390 @@
+/*
```

Comment:
A bunch of the documentation in this file is _very_ Matter.framework-specific.  That should probably be fixed.

Reply:
updated, thanks